Server/VikingVault.Services/BankAccountService.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using VikingVault.DataAccess;
+using VikingVault.DataAccess.Models;
+using VikingVault.Services.Abstractions;
+using VikingVault.Services.Exceptions;
+
+namespace VikingVault.Services
+{
+    public class BankAccountService : IBankAccountService
+    {
+        private readonly VikingVaultDbContext _dbContext;
+
+        public BankAccountService(VikingVaultDbContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+
+        public BankAccount createBankAccount(BankAccount account)
```

Comment:
This method name doesn`t follow the C# naming convention. PascalCase should be used instead.